keen/slate_algolia
# Add rspec tests for the extension file

## What does this PR do?

Adds a structure for rspec unit tests, as well as some very basic tests for the main extension file
## How should this be tested?

Pull down the PR and:

``` sh
bundle install
bundle exec rspec
```

That should test both that I actually _did_ set up rspec, and also that my tests pass :)
